orchestra/static/orchestra/todos/todo-checklist.html

```diff
@@ -5,7 +5,7 @@
            ng-model="todo.completed"
            ng-change="updateTodo(todo)">
     <label ng-class="{'text-danger': isInDanger(todo)}">
-      <span ng-click="toggle(this)">
+      <span ng-click="toggleTodo(todo, this)">
```

Comment:
two things:
1) My mind is cloudy, but why is it possible to toggle a checklist item from two places (`todo-checklist.html` and `todo-list.html`
2) Should this file be called `todo-list-item.html`?

Reply:
1. The functionality to toggle is implemented in `todo-checklist` whereas functionality of what happens when a todo is toggled is implemented in `todo-list`.
2. This file not only renders the entire todolist in a checklist format and hence the name `todo-checklist.html`